03-Node-npm-modules/Director.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Director module
+ */
+module.exports = (function (name) {
+  var attributes = {};
```

Comment:
Is best practice to add a 'use strict' pragma, globally for the whole module, or at the beginning of the function.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode